app/assets/javascripts/vendor/lib/google/maps.js.erb

```diff
@@ -1,419 +1,417 @@
+          canvas_ratio = width / height;
         }
       }
-    }
-  },
-
-  log_2: function (x) {
-    return Math.log(x) / Math.LN2;  // log_2(x) = ln(x)/ln(2)
-  },
-
-  get_window_center: function (bounds) {      // for use with home-brew geoJSON scanner/enumerator
-    var xminp = bounds.xminp;           //
-    var xmaxp = bounds.xmaxp;           //
-    var xminm = bounds.xminm;           //
-    var xmaxm = bounds.xmaxm;           // these elements already set
-    //if (bounds.prime /*&& bounds.anti*/) {
-    //  xminp = 0;
-    //  xmaxm = 0;
-    //}
-    //if (bounds.anti) {
-    //  //xmaxp = 180;
-    //  if(xminm <= -180) {
-    //    if ((xminm + 360) < xmaxp) {
-    //      xminm = xminm + 360;
-    //    }
-    //
-    //  }
-    //}
-    //
-    var ymin = bounds.ymin;             //
-    var ymax = bounds.ymax;             //
-
-    var center_long = bounds.center_long;   // these elements are calculated here,
-    var center_lat = bounds.center_lat;     // unless preset validly in data-map-center
-    var gzoom = bounds.gzoom;               //
-
-    if (center_long == undefined) {    //determine case of area extent
-      center_long = 0.0;
-      var wm = 0.0;        // western hemisphere default area width
-      var wp = 0.0;        // eastern hemisphere default area width
-      if ((xminp == 180.0) && (xmaxp == 0.0)) {    //if no points, null out the range for this hemisphere
-        xminp = 0.0;                                // possible place to mark singleton point
+      var bounds = {};    //xminp: xmaxp: xminm: xmaxm: ymin: ymax: -90.0, center_long: center_lat: gzoom:
+      this.getData(chained, bounds);               // scan var data as feature collection with homebrew traverser, collecting bounds
+      bounds.canvas_ratio = canvas_ratio;
+      bounds.canvas_width = width;
+      bounds.canvas_height = height;
+      var center_lat_long = this.get_window_center(bounds);      // compute center_lat_long from bounds and compute zoom level as gzoom
+      map.setCenter(center_lat_long);
+      map.setZoom(bounds.gzoom);
+      //map.fitBounds(bounds.box);  /// no better results on non 2:1 canvas ratios
+      if (document.getElementById("map_coords") != undefined) {
+        document.getElementById("map_coords").textContent = 'LAT: ' + center_lat_long['lat']() + ' - LNG: '
+          + center_lat_long['lng']() + ' - ZOOM: ' + bounds.gzoom;
       }
-      if ((xmaxm == -180.0) && (xminm == 0.0)) {    //if no points, null out the range for this hemisphere
-        xmaxm = 0.0;                                // possible place to mark singleton point
+      var sw = bounds.sw;       // draw the bounding box for JDT
+      var ne = bounds.ne;
+      var coordList = [];
+      coordList.push([sw['lng'](), sw['lat']()]);   // southwest point
+      coordList.push([sw['lng'](), ne['lat']()]);   // northwest point
+      //if (sw['lng']() > 0 && ne['lng']() < 0) {     // are we spanning the prime meridian
```

Comment:
This is related to the deferred back-integration of anti-meridian handling into the maps code.  Indubitably this should be done at some point, but seemed low priority when deferred.